configs/second/metafile.yml

```diff
@@ -13,6 +13,21 @@ Collections:
       URL: https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/backbones/second.py#L11
       Version: v0.5.0
 
+  - Name: SECOND-FP16
+    Metadata:
+      Training Techniques:
+        - AdamW
+        - Mixed Precision Training
+      Training Resources: 8x TITAN Xp
+      Architecture:
+        - Hard Voxelization
+    Paper:
+      URL: https://arxiv.org/abs/1710.03740
```

Comment:
Should we use the paper link of FP16? I thought FP16 models should use the same collection with different training techniques.